src/common/client.ts

```diff
@@ -368,18 +373,29 @@ export async function getFallbackImageCollection(): Promise<SanityImage[]> {
 }
 
 /**
- * Get legal text for footer
- * @todo We probably want to give the footer it's own content type in the future
+ * Gets legal and copyright text for footer from org
+ *
+ * @returns
  */
-export async function getSmallPrint(): Promise<PortableText> {
+export async function getFooterData(): Promise<FooterData> {
   const client = getClient();
-  const query = groq`*[_id == "organization-singleton"][0].smallPrint`;
+  const query = groq`*[_id == "organization-singleton"][0]{
```

Comment:
See other comment - move this into `model/footer` then use directly in fetch method.

See `getAboutPageData` function below for example

Reply:
ok thanks will do